### PR TITLE
remove folly workaround

### PR DESCRIPTION
**Pull request by machard**
https://github.com/facebook/flipper/issues/2215

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -54,17 +43,4 @@ target 'ledgerlivemobile' do
...
-    replace = text.gsub(findstr,replacestr)
-    if text != replace
-      puts "Fix: " + name
-      File.open(name, "w") { |file| file.puts replace }
-      STDOUT.flush
-    end
-  end
-  Dir[dir + '*/'].each(&method(:find_and_replace))
-end
+end
```
Comment (juan-cortes) [inline ios/Podfile:46]: new line please, we are civilised here

**Review by juan-cortes — DISMISSED** [review]

**Review by valpinkman — DISMISSED** [review]

**Comment by rjarasson-ledger** [comment]
I tested locally, and it fixed my issue not being able to build on MacOS BigSur.

File: ios/ledgerlivemobile.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -514,10 +514,12 @@
 			inputPaths = (
 				"${PODS_ROOT}/Target Support Files/Pods-ledgerlivemobile/Pods-ledgerlivemobile-frameworks.sh",
 				"${PODS_ROOT}/../../node_modules/@ledgerhq/react-native-ledger-core/ios/Frameworks/universal/ledger-core.framework",
+				"${PODS_XCFRAMEWORKS_BUILD_DIR}/OpenSSL/OpenSSL.framework/OpenSSL",
 			);
 			name = "[CP] Embed Pods Frameworks";
 			outputPaths = (
 				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/ledger-core.framework",
+				"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/OpenSSL.framework",
```
Comment (gre) [inline ios/ledgerlivemobile.xcodeproj/project.pbxproj:522]: is there a reason we suddenly need OpenSSL?

Comment (gre) [reply ios/ledgerlivemobile.xcodeproj/project.pbxproj:522]: oh it's for flipper?

**Pull request merged**